Lib/test/test_importlib/extension/test_loader.py

```diff
@@ -29,6 +29,25 @@ def test_load_module_API(self):
         with self.assertRaises(ImportError):
             self.load_module('XXX')
 
+    def test_type_error(self):
+        with self.assertRaises(TypeError) as cm:
+            other = self.machinery.ExtensionFileLoader(
+                fullname=util.EXTENSIONS.name,
+                name=util.EXTENSIONS.name,
+                path=util.EXTENSIONS.file_path)
+
+        self.assertIn("name", str(cm.exception))
+        self.assertIn("fullname", str(cm.exception))
+
+    def test_name_deprecation(self):
```

Comment:
It would be better if we have a more descriptive test name here. Specifying only `name` makes it a bit unclear.